docs/getting-started/first-abn.md

```diff
@@ -164,12 +144,20 @@ svc/backend deploy/backend \
 svc/backend-candidate-1 deploy/backend-candidate-1
 ```
 
-Delete the application description:
+Delete the application routemap:
 
 ```shell
-kubectl delete cm/backend
+cat <<EOF | helm template routing --repo https://iter8-tools.github.io/iter8 routing-actions --version 0.18 -f - | kubectl delete -f -
```

Comment:
This `helm template` into `kubectl delete` is so strange to me.

Reply:
It avoids us making any assumptions about what resources were created. The interface (and inputs) are the same as create for the user.